data/scripts/eventcallbacks/player/on_look.lua

```diff
@@ -60,11 +60,12 @@ function callback.playerOnLook(player, thing, position, distance)
 				description = string.format("%s\nDecays to: %d", description, decayId)
 			end
 		elseif thing:isCreature() then
-			local str = "%s\nHealth: %d / %d"
+			local str, pId = "%s\n%s\nHealth: %d / %d"
 			if thing:isPlayer() and thing:getMaxMana() > 0 then
+				pId = string.format("Player ID: %i", thing:getGuid())
 				str = string.format("%s, Mana: %d / %d", str, thing:getMana(), thing:getMaxMana())
 			end
-			description = string.format(str, description, thing:getHealth(), thing:getMaxHealth()) .. "."
+			description = string.format(str, description, pId, thing:getHealth(), thing:getMaxHealth())
```

Comment:
I didn't quite understand this "pId"

Reply:
> I didn't quite understand this "pId"

"Set PlayerID of accesses players on on_look script. (usable to know the real player ID)."